### ISSUE TITLE
do the gateware support 3d lut ?

### ISSUE THREAD

**Issue by IngenicC** [apertus-open-source-cinema/axiom-firmware#178]
hello!
do the gateware support 3d lut ?

**Comment by anuejn** [comment]
No, currently it does not support applying 3d luts to the output. However some color conversion is supported by a matrix style lut and 1d luts for the individual channels
